packages/canary/src/icons/harness-logo.svg

```diff
@@ -0,0 +1,14 @@
iu/I6FNVvUzbkYpZouelhAiOa0XIRqQ9LtCAMK6686K68tiN6I21b7rErry5ENlr8lDa5nLp4aC7UGZ5wgY2XjdeBRTlaZVwWqlvSEYZNxb3lNxvRS4jWZuNAXLme8kGXUL+mj+lhfCeq0YKfhDFaELYVV56q0Jg8C+QeorTyUnbg6A125bWXZalxOLqEaEUrC/yUci4PY7QgbCuuPBGlOvGLyK28hAF25bWXZalxSORWWhngfjLZcN2xvINwrrjy7MoruiuvfLSEU02P4qEMyv1k+14V9vtohnPFlRf1lVfZoEsjufKKarREl0YhWhD2cEHHeKVNj+MZYQAq10UuXsuT/4LoA5GMVlTuWE7YNxX3pio0JOsQmUfUNhtFbySzZW7oNxuXJU8mxlKUk0yP4qFWYDGZvtdHJVoQpXDB3vG6B6g0PY5nohCvqEZL5R4Gyg1MqNxhehgvRStcEO14Id8lm7mTuqpdpscpKhutSEULohiuvOXNZ0UuXrAbuItM9ubQxGtZ8mQc7gc+ZHoUD7WCzKVcb4jqlXCjG66orrxgN8qdZLO3BD5e0YxWC8g9DJAbo7jSyotuuPLsyiuY8YpmtHYicjtlekdUV1p5NlyqQlP6c6jOw8YrGJanPrzniPiIRYtbqdhxBxPHtpgexjQbrrzGVF0E49UC3BmoeEV2pWWjtbfwH4DaXVMrH0bkYqJ0kCpUAJdTFruWVekBpofpko2WjdYedsXVUWRXXvo9Ms53fHuxuRXN7ycrPwI+YnoUD23bE627bbT2ZcPVmVy87gZGmB7FQy7IYzhcxdT4X0wPs4/65CjKuB/4tOlRPPQuwvW8FV/IdGkzPYzf2HAdSFPzZFyZD4w2PYrHXkK4nrb4o9RJ1vQwLFs3Fic2Dzid6Lxec9EaFF/AeMmYHsaPovJC6Jnlqc8iuoDoxWszsIRYZr7RmyssT38M3LsidsK0jVY32HB1ZVlqIo4uJHrxUpBnQe9ka8tKzh2z27NnfvD1vgzq+3lcvR5hjGfPa56NVjfZcHVHdOMFsA14BEcX0pp4ruSbj03p43H1UtCzgP4lfS5/2YRyNUPiD9hodc2Gq7uiHS+At4AGHP0xrVueL/pVJho3HI3rng16bsRWWQCbEP0mLyQeZI64pocJAhuuQth4AbwN8l+I/oIynu7Vdc3r3xpIn7YTyDpTEJ0KjCV6r0kbrR6I2ouk93LxWgAcaXoUw3YhrEFZhcjvcDKvUt7/LfoN2dnpps5qLWPXpv60ZIaBewyu+1FE/oPcwaSDPZ/eHzaBXsGLiR/YaBXGhqsn7Mqro93ARpQkQjOwEfRdkCwqDqKHogxDdAQiCZQjgL6GZzbtLdAreTHxQxutwtlw9VRj6jOoLsSuvKzCpVG9hJpEPSJqepggsuHqjVy8FmFXXlb32WgVgQ1Xb9mVl9V9aUQupjpeb3qQoLNXh+it6vgvEZkJvGF6FMvXUohczNTKh00PEgY2XMVg42UdjLAWZRbVcbt5WCR2U7GY7Gajtb83UZlFbfxx04OEiQ1Xsdl4We1stErEbioWW3X8l8CFKK+bHsUyykarhGy4SqEm8SsEu88ruv6JKzNttErHbiqWUkPydJSFETxpOMpeQWUGtfHVpgcJMxuuUmtIng4swu7zioJXEJlOdfwp04OEnd1ULLWaxK+w+7yiwEbLQzZcXsjv87LxCqu/22h5y4bLKzWJX6F25RU+8iJwvo2Wt+w+Lq8tS05AWGR32IfCC4hMozr+e9ODRI0Nlwk2XmFgo2WQDZcpNl5BZqNlmN3HZcqZiVV2n1cACX+y0TLPhsukfLzgn6ZHsbpDnyWb/aqNlnl2U9EP7GZjAOizOLFpTK18yfQklg2XfzQmP40yHzjG9ChWB8pziHM+NZUvmx7FyrGbin5RnXgC+BrwgulRrH38GZhpo+UvNlx+UpN4GtyvAL81PYpFbke8yjRqE380PYq1L7up6EePpMeQcW8HqoGY6XGiSZ6B7IXUjLQrYB+yKy4/OqPydXBnAPOAnabHiaBfg37NRsu/7IrLz1a+WkHLwPNQrgVNmB4nArKILEPcK5g6Yp3pYawDs+EKgob0aah7C8JHTY8SYjsRFtGWvYW6qk2mh7EOzoYrKBpTR+LqtQhnA/1MjxMyKVRuot/2B5g4tsX0MFbXbLiCpH5dP8pi5wDfAo42PU4oKM/huNdQPfJJ06NY3WfDFUSN6z+EylUgU4AK0+ME1C6Uhyh3bs59GGIFiQ1XUK3YeAiZtrMRLgXeb3qcgHkF0dtpcx+irmqX6WGswtlwBd2y5AdwuAQ4Cxhkehyf24mwHHVut0fCB5sNVxgs0XKGpT4FXAyMx24+dqTA86B3k3FX2FVW8NlwhUn9psMob6nG1RkIJ2OPugd4A+RBNPt9akeuNz2MVRw2XGG0YsNwstmzEL6KcgLRDNgGVB4mJkvtpWjCx4YrzOrXjSDm1CByDnASUG56pNKTJOijOM6DtA5/njrJmp7IKj4brihoSB8BejroF4FTgENMj1RkCrwGugKVn5GNv2CDFW42XFGyYuMhuG2n4FKH8J/AKIL9GtgO/AlYhsQep3r4P0wPZHkjyC9aq6fqNUZZ89HgnA5MAv4VGGJ4qu5qBV5D5EnUfRRp/QPVY7aYHsrylg1X1K1KD2A7xyPup0DGgx4HDMNflzzaDvwD5X9xeJK27LPUVTWbHsoyx4bLarcqPYBtbUcTcz6MK6cgnAQcSe7AVu9CpuxASIL8FdXnQJ5BZA3Vwzciop7NYfmWDZfVOVXh4fWDKY+NAT6IK8cj+n5gNLkV2WHkDnTt6SeVWXKbfTuAd0DWg76K6BpcZw0x9zVat2ygblxrMf44VrjYcFndt/LVCtoOHUS2rRI3NgJHRwEjQeLgHgEyGOQQ0D4I/VEccnHaRe5KrlsQNqKkUUkirEVlHbgpBjpvM6Fyh9E/n2VZlmWVyv8HWLAsP5MtI6MAAAAASUVORK5CYII="/>
```

Comment:
this is not a real svg.. it seems to be base64 encoded into an svg. 
also, we should generally keep the harness logo independent of the background circle. 
we can pick the image from https://github.com/harness/uicore/blob/main/packages/icons/src/harness.svg

Reply:
This should not be a merge blocker.